oval-schemas/windows-system-characteristics-schema.xsd

```diff
@@ -798,9 +798,9 @@
      <xsd:element name="ntuser_item" substitutionGroup="oval-sc:item">
           <xsd:annotation>
                <xsd:documentation>To ensure consistent results across OVAL interpreters should comply with the following:</xsd:documentation>
-			  <xsd:documentation>* Create a 'does not exist' item for each ntuser.dat file that does contain the required registry key.</xsd:documentation>
+			  <xsd:documentation>* Create a 'does not exist' item for each ntuser.dat file that does not contain the specified registry key.</xsd:documentation>
 			  <xsd:documentation>This requirement goes against normal item creation in OVAL, but is very useful in reporting to the end user which ntuser.dat files failed registry key existence checks when tests mandate 'all_exist'</xsd:documentation>
-			  <xsd:documentation>Example:  If a computer contains 3 user profiles of human users, and one user has the registry key/name/value required, and is configured correctly, but the other 2 users do not have the required registry key, creating 'does not exist' items, informs the end user, which ntuser.dat files, (which corresponds to which specific user(s)) are not compliant.</xsd:documentation>
+			  <xsd:documentation>Example:  If a computer contains 3 user profiles of human users, and one user has the registry key/name/value specified, and is configured correctly, but the other 2 users do not have the required registry key, creating 'does not exist' items, informs the end user, which ntuser.dat files, (which corresponds to which specific user(s)) are not compliant.</xsd:documentation>
```

Comment:
good catch again.  Will fix

Reply:
and back to your "any domain user with an ntuser.dat file on a machine must have logged on at some point' statement.  6 months ago I would have agreed, but had end users share ntuser.dat files that proved otherwise.  I diffed their stub ntuser.dat file which was created by some 3 party application that spawns commands and returns data back to a remote computer, and found that there can be ntuser.dat files that are residue from that process.  It might also be possible create stub ntuser.dat files with remote powershell commands based on options provided as well.

In my comparison of a 'normal' ntuser.dat file and the stub file from an end user, is where I found the differences with the Explorer and "UserSignedIn" value.   The value doesn't exist at all with this scenario, and is the basis of this documentation update based on our implementation.

Reply:
Interesting.  I bet these are artifacts from services that are configured to run using domain accounts.

I know in the past I've seen these as residue from bad image-based installations with bad sysprep, but it's been a long time, and I always considered that a real hygiene problem that customers had and should fix.